test/verify/check-storage-lvm2

```diff
@@ -283,6 +283,12 @@ class TestStorageLvm2(StorageCase):
         b.wait_in_text("#drives", disk)
 
         m.execute("vgcreate TEST %s" % disk)
+
+        # HACK - Sometimes the vgcreate call above doesn't update
+        #        udev.  Make sure it happens.
+        #
+        m.execute("udevadm trigger")
```

Comment:
That is a really big and blunt hammer, and I wouldn't be suprised if that messes up other things in the VM. `--name-match {disk}` or at least `--subsystem-match block` would be more targetted. But I see that we commit this sin a lot in the tests, so it doesn't matter that much if we clean up 5 or 6 places.

Reply:
If we are worried about that, we should only trigger /dev/sda via `echo change >/sys/block/sda/uevent` or similar I guess.